doc/contributing/releases.md

```diff
@@ -249,6 +249,14 @@ each commit from the PR. You will also need to manually apply the PR
 metadata (`PR-URL`, `Reviewed-by`, etc.) by amending the commit messages. If
 known, additionally include `CVE-ID: CVE-XXXX-XXXXX` in the commit metadata.
 
+Push the commits to the `vN.x-staging` branch and create the proposal branch
```

Comment:
FWIW, I haven't always used the private staging branch when preparing security releases. I'd instead build up a release directly in a proposal branch that has branched off of `vN.x`, such as:

```
git checkout v18.x
git checkout -b v18.x.x-proposal
git cherry-pick  ...  # cherry-pick nodejs-private PR commits directly into the proposal
# create release commit
```

The reason I sometimes preferred that method was because the public staging branch often may already have regular commits on it, and this way avoids having out-of-sync staging branches between public and private. We'd have the proposal branch containing `v18.x` content + the security release commits. The `upstream/v18.x-staging` branch gets rebased at the end of the release process (as per usual) to pick up the security commits once they've been merged into `upstream/v18.x`

Both ways work, though. But, I liked the easy confirmation this method gave that the security release proposal definitely just included the security patches on top of the latest release (`v18.x`).

Reply:
> The reason I sometimes preferred that method was because the public staging branch often may already have regular commits on it

When does it happen? Because, before creating the proposal the releaser should make sure the v18.x and v18.x-staging are up-to-date with the `v18.x`. For instance:

```console
git checkout v18.x
git reset --hard upstream/v18.x
git push private v18.x-staging
git push private v18.x
```

would make the v18.x-staging and v18.x similar on the private repo.

Reply:
> The reason I sometimes preferred that method was because the public staging branch often may already have regular commits on it, and this way avoids having out-of-sync staging branches between public and private.

I'm referring to it avoiding `upstream/v18.x-staging` and `private/v18.x-staging` diverging.